### ISSUE TITLE
[Bug] Cinematic Camera disables after zoom

### ISSUE THREAD

**Issue by AlNexus** [sp614x/optifine#3802]
## Description of Issue
Alright, so what happens is: I turn on Cinematic Camera, everything works fine but if I zoom it will continue on but when I stop zooming it disables cinematic camera, I think it's a simple bug, since you just need to determine if the Cinematic Camera will be disabled or not, basically:
Cinematic Camera ON (Player Toggled) > Zoom In > Enable Cinematic Camera > Zoom Out > If "Player Toggled" is false > Disable Cinematic Camera
So if the cinematic camera was toggled on by the player, it won't stop the cinematic camera when releasing the zoom key

## Steps to Reproduce
Turn on Cinematic Camera, Zoom in, release zoom, cinematic camera is now off, even tho it was supposed to be on since the player toggled it on.

## OptiFine Version
All of them.

## Installation Method
I installed OptiFine using the jar file, in which it install a profile.

- Does this happen in Vanilla without OptiFine?
No.

## Additional Information
Basically, it's just optifine, a small flaw, I hope this gets fixed, not sure if it's a bug but it really **bugs** me.

**Comment by Goncorer** [comment]
This is because zooming in automatically turns on the the cinematic camera, and turns it off when you stop zooming. It's just a matter of checking if it was enabled before zooming to not disable it

**Comment by naeqirelle2020** [comment]
Still Occurs in latest preview (Pre-19)
